ruby/ql/lib/codeql/ruby/frameworks/GraphQL.qll

```diff
@@ -0,0 +1,270 @@
+private import codeql.ruby.AST
+private import codeql.ruby.Concepts
+private import codeql.ruby.controlflow.CfgNodes
+private import codeql.ruby.DataFlow
+private import codeql.ruby.dataflow.RemoteFlowSources
+private import codeql.ruby.ast.internal.Module
+
+
```

Comment:
We don't (yet) want users to rely on these classes, as they may change in the future. So let's wrap everything in a `private module`.

```ql
private module GraphQL {
  ...
}
```

We can then drop the `private` annotations from the individual classes.